docs/advanced/append-nodes.md

```diff
@@ -0,0 +1,50 @@
+# Append additional nodes to the builder
```

Comment:
I think this should do a quick explainer about "buildx also supports running builds on multiple machines" and how the builds are split up then.